jsjaws.py

```diff
@@ -99,14 +133,26 @@ def stop(self) -> None:
         self.log.debug("JsJaws service ended")
 
     def execute(self, request: ServiceRequest) -> None:
+        request.result = Result()
         self.artifact_list = []
+        file_to_analyze = request.file_path
+
+        if request.file_type == "code/html":
```

Comment:
We also want to be able to handle code/hta files https://github.com/CybercentreCanada/assemblyline-service-extract/blob/47f63a0241ea7fbddc00093f886a47fe17d814ed/extract/extract.py#L187